### PR TITLE
Don't use deprecated EventListener class.

### PR DESCRIPTION
**Pull request by ehsannas**
I have separated the PR into 2 commits. The first is only formatting changes applied by clang-format. The second is the one needed for a successful compilation since EventListener has been removed from the public API.

### CODE REVIEW

File: firestore/testapp/src/common_main.cc [hunk 1]
```diff
@@ -25,14 +25,14 @@
...
-#include "main.h"  // NOLINT
+#include "main.h" // NOLINT
 
 const int kTimeoutMs = 5000;
 const int kSleepMs = 100;
 
 // Waits for a Future to be completed and returns whether the future has
 // completed successfully. If the Future returns an error, it will be logged.
-bool Await(const firebase::FutureBase& future, const char* name) {
+bool Await(const firebase::FutureBase &future, const char *name) {
```
Comment (var-const) [inline firestore/testapp/src/common_main.cc:35]: We are attaching asterisks and ampersands to the type name, not the variable name in our code (including the public headers and the documentation), so we should follow the same style here. `clang-format` is likely configured to use a heuristic to apply the style prevalent in the file.

Is there a `.clang-format` file in the repo? If not, you might want to add one (just copy our existing one from the [C++ repo](https://github.com/firebase/firebase-cpp-sdk/blob/main/firestore/.clang-format)). The relevant settings are `DerivePointerAlignment` (whether to apply the heuristic, should be `false`) and `PointerAlignment` (should be `Left`).

**Review by var-const — CHANGES_REQUESTED** [review]

File: firestore/testapp/src/common_main.cc (hunk 1 above)
Comment (ehsannas) [reply firestore/testapp/src/common_main.cc]: Done

Comment (var-const) [reply firestore/testapp/src/common_main.cc]: If you used a custom `.clang-format` file, it's probably a good idea to check it in.

**Review by var-const — APPROVED** [review]

File: firestore/testapp/src/common_main.cc (hunk 1 above)
Comment (ehsannas) [reply firestore/testapp/src/common_main.cc]: ah, I meant to do that and forgot. I'll send another PR for it.

**Pull request merged**